packages/workflow/package.json

```diff
@@ -2,27 +2,29 @@
   "name": "@temporalio/workflow",
   "version": "0.17.2",
   "description": "Temporal.io SDK Workflow sub-package",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
-  "scripts": {},
   "keywords": [
     "temporal",
     "workflow",
     "isolate"
   ],
-  "author": "Roey Berman <roey@temporal.io>",
-  "license": "MIT",
-  "dependencies": {
-    "@temporalio/common": "file:../common",
-    "@temporalio/proto": "file:../proto"
-  },
+  "homepage": "https://github.com/temporalio/sdk-typescript/tree/main/packages/workflow",
   "bugs": {
     "url": "https://github.com/temporalio/sdk-typescript/issues"
   },
-  "homepage": "https://github.com/temporalio/sdk-typescript/tree/main/packages/workflow",
+  "license": "MIT",
+  "author": "Roey Berman <roey@temporal.io>",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "files": [
     "lib"
   ],
+  "scripts": {},
+  "dependencies": {
+    "@temporalio/workflow-common": "file:../workflow-common"
+  },
+  "devDependencies": {
+    "@temporalio/proto": "file:../proto"
```

Comment:
Is this a dev dependency now? Is it even a dependency anymore?

Reply:
I added as devDep because we `import type { coresdk }` in a few places in this package

Reply:
It should be a non dev dependency if we export any of those types or anything that uses those types (e.g. function signatures), to be safe, let's make it non dev.